packages/core/src/sass/currency-select-radio.scss

```diff
@@ -120,3 +121,25 @@
         width: calc(100% + 2.4rem);
     }
 }
+
+.currency-selector {
+    &__radio-group--with-margin {
+        &--is-header {
+            font-size: 1.6rem;
+            margin: 2rem 0 0;
+            font-weight: bold;
+            color: var(--text-prominent);
+        }
+
+        @include desktop {
+            margin-bottom: 6rem;
+            margin-left: auto;
+            margin-right: auto;
+            width: 704px;
+        }
+
+        @include mobile {
+            margin-bottom: 7rem;
+        }
+    }
+}
```

Comment:
Is there any reason that you didn't update the existing code in `add-or-manage.scss`?